app/assets/stylesheets/application.scss

```
@@ -13,3 +13,5 @@
  *= require_tree .
  *= require_self
  */
+@import "bootstrap-sprockets";
+@import "bootstrap";
```

thêm end-line vào
tức là line trống ko chứa j kể cả space
```suggestion
@import "bootstrap";

```